config/environments/production.rb

```diff
@@ -76,6 +76,9 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
+  # Prevents schema dump from throwing error in production.
+  config.active_record.schema_format = :ruby
```

Comment:
We cannot accept this. The database of catarse today has several features not supported by the schema.rb
The rake db:migrate should work in heroku and the error it usually will usually report is safe to ignore:
```
rake aborted!
Error dumping database
```